---
layout: default
title: Nielsen Lab / research
categories:
 - research
---

    <h1>Admixture Graphs</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/Picture1.png" style="float:right">

     Admixture graphs provide a  
     concise description of the historical demographic relationships between  
     genetic samples of populations, assuming their relationships are the  
     product of discrete, instantaneous splits and admixture events. Each 
      admixture graph topology is associated with parameters capturing genetic drift 
       and admixture proportions, and once these are fitted to genetic data, 
       the goodness of fit can be measured to determine how accurately the 
        graph captures the historical relationship between samples. Inferring 
         graph topologies, however, involves a combinatorial search, and  
         since the space of graphs grows super-exponentially in the number 
          of populations and the number of admixture events, an exhaustive  
          search is typically not possible. Instead, the search for well-fitting  
          topologies is often done manually or through greedy algorithms, which  
          are not guaranteed to find a global optimum.  This motivated the development of 
           a novel MCMC sampling method,  <a href="https://github.com/avaughn271/AdmixtureBayes">AdmixtureBayes</a>,  
           that can sample from the posterior distribution of admixture graphs.  
           This enables an effective search of the entire state space as well as  
           the ability to report a level of confidence in the sampled graphs.  
            We are actively looking at ways to improve AdmixtureBayes  by using 
             more information in the data than just the  
             covariance in allele frequencies between populations. 

     <div class="bigspacer"></div>

     <h4>Example Papers</h4>
        <ol type = "1">
          <li><a href="/pdfs/papers/disentangle.pdf">Disentangling selection on genetically correlated polygenic traits via whole-genome genealogies</a></li>
         </ol>
    <div class="bigspacer"></div>

    <h1>Ancestral Recombination Graphs</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/Picture2.png" style="float:right">

     Ancestral recombination  graphs (ARGs) represent the history of coalescence and 
     recombination events in the history of a sample of DNA sequences. There is currently 
      significant interest in  both the  development of  methods to infer ARGs from sequence 
      data and the application of ARGs to estimate population genetic parameters. My research 
      has focused mostly on the latter, specifically the development of methods to infer 
      demographic history and natural selection from DNA using ARGs. I am the lead developer and 
      current maintainer of the  <a href="https://github.com/avaughn271/CLUES2">CLUES2</a> software. 
      CLUES2 can infer time-varying selection coefficients, selection under arbitrary dominance 
      scenarios, and can work with ARGs inferred on ancient DNA. I also have an ongoing project on using 
      ARGs to infer patterns of human migration and dispersal.  
      
     <div class="bigspacer"></div>
     <h4>Example Papers</h4>

     <ol type = "1">
      <li><a href="/pdfs/papers/fastadm.pdf">Fast admixture analysis and population tree estimation for SNP and NGS data</a></li>
      </ol>
 <div class="bigspacer"></div>

       <h1>Analyses of Ancient DNA</h1>
       <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
       <img src="/images/Picture4.png" style="float:right">

       I work closely with Eske Willerslev and other researchers at
       the <a href="https://globe.ku.dk/research/lundbeck-foundation-geogenetics-centre/">Lundbeck  
        Foundation Center for GeoGenetics</a> at the University of Copenhagen
        on analyzing ancient human DNA. Specifically, I have worked on studying ancestry-specific 
        selective pressures in Stone Age Eurasian populations and quantifying the ability 
        to accurately infer ARGs on low-coverage data. I have also re-analyzed several 
        datasets generated by the  Center for GeoGenetics for both the CLUES2 and  
        AdmixtureBayes projects, generating new insights into changing selective pressures on 
        diet-associated alleles and the peopling of the Americas respectively.

          <div class="bigspacer"></div>
          <h4>Example Papers</h4>

          <ol type = "1">
            <li><a href="/pdfs/papers/viking.pdf">Population genomics of the Viking world</a></li>
           </ol>
      <div class="bigspacer"></div>

      <h1>Human Adaptation and Evolution</h1>
      <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
      <img src="/images/Picture1.png" style="float:right">
  
      TBD
  
       <div class="bigspacer"></div>
  
       <h4>Example Papers</h4>
          <ol type = "1">
            <li><a href="/pdfs/papers/joint.pdf">Joint Estimation of Pedigrees and Effective Population Size Using Markov Chain Monte Carlo</a></li>
           </ol>
      <div class="bigspacer"></div>
  